auto_comps.py

```diff
@@ -0,0 +1,224 @@
+"""
+Auto comps loader WompWomp99 (with autocache)
+Champion names are different in lolchess/communitydragon/tftactics
+Some champion names is repeating because of different patches (but it's the same champion), for example Nomsy
+In-game champion names only in communitydragon
+"""
+import re
+from urllib import parse
+
+import game_assets
+import json
+import requests
+from difflib import SequenceMatcher
+import os
+from comps import CompsManager
+from bs4 import BeautifulSoup
+
+LOLCHESS_CHAMPIONS_URL = 'https://lolchess.gg/champions/'
+LOLCHESS_META_COMPS_URL = 'https://lolchess.gg/meta'
+DRAGON_URL = 'https://raw.communitydragon.org/latest/cdragon/tft/en_us.json'
+
+HARD_OVERRIDE_LIST = {'TFT7_Wukong': 'Wukong'}
+LOLCHESS_BOARD_ARRANGE = [21, 22, 23, 24, 25, 26, 27, 14, 15, 16, 17, 18, 19, 20, 7, 8, 9, 10, 11, 12, 13, 0, 1, 2, 3,
+                          4, 5, 6]
+
+def Parse(input, fromwhere, to, startindex=0):
+    index_from = input.find(fromwhere, startindex)
+    if index_from != -1:
+        index_to = input.find(to, index_from + 1)
+        if index_to != -1:
+            return (int(index_from), input[index_from + len(fromwhere):index_to])
+    return (-1, None)
+
+
+def ParseMultiple(input, fromwhere, to):
+    output = []
+    index = 0
+    while index != -1:
+        current = Parse(input, fromwhere, to, index)
+        index = current[0]
+        if index != -1:
+            output.append(current)
+            index += len(fromwhere)
+    return output
+
+
+# def __LoadLolChessCurrentSetVersion(input_str):
+#     return Parse(input_str, 'lolchess.gg/champions/set', '"')[1]
+
+def __LoadLoLChessPrices():
+    lolchess_url = LOLCHESS_CHAMPIONS_URL  # + set_input+ '/'
+    response = requests.get(lolchess_url)
+    current_tft_set = response.url.split('/')[4][3:]
+
+    json_in_text = Parse(response.text, 'window.guideChampion = {\n            champions: ', r'}}],')[1] + r'}}]'
+    json_parsed = json.loads(json_in_text)
+
+    output_dictionary = {}
+    for each_character in json_parsed:
+        character_code = each_character['code']
+        character_name_ingame = each_character['ingame_code']
+        character_price = int(each_character['cost'])
+        if character_name_ingame not in output_dictionary:
+            output_dictionary[character_name_ingame] = (character_price, character_code)
+    return [current_tft_set, output_dictionary]
```

Comment:
Some swapping/clean up here
```suggestion
    output_dictionary = {}
    for each_character in json_parsed:
        character_code = each_character['code']
        character_name_ingame = each_character['ingame_code']
        character_price = int(each_character['cost'])
        if character_name_ingame not in output_dictionary:
            character_code = each_character['code']
            character_price = int(each_character['cost'])
            output_dictionary[character_name_ingame] = (character_price, character_code)
    return [current_tft_set, output_dictionary]
```